From 3c03e4b81c9de73e57a56029f7f8cda8bec24e91 Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@openwide.fr>
Date: Sat, 10 Jan 2015 15:48:21 +0100
Subject: [PATCH] configure.in: remove unsafe headers path

Forcing /usr/local/include headers path in CFLAGS is unsafe for
cross-compilation.

Signed-off-by: Romain Naour <romain.naour@openwide.fr>
---
 configure.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.in b/configure.in
index a8fc950..93a4b77 100644
--- a/configure.in
+++ b/configure.in
@@ -82,7 +82,6 @@ AM_CONDITIONAL(CROSS_COMPILING, test x$cross_compiling = xyes)
 dnl ***********************************************************************
 dnl set compiler/linker flags
 dnl ***********************************************************************
-INCLUDES="${INCLUDES} -I/usr/local/include"
 AC_SUBST(INCLUDES)
 SANE_SET_CFLAGS([$is_release])
 SANE_SET_LDFLAGS
-- 
1.9.3

